src/tools/convert/topology_helper.py

```diff
@@ -58,7 +58,12 @@ def FillFileBuffer(fname, filebuffer):
     for line in f:
 	if "include" in line and not line[0]==';':
 	    name=(line.split()[1]).strip('\"')
-	    FillFileBuffer(name, filebuffer)
+            try: 
+ 	        FillFileBuffer(name, filebuffer)
+            except IOError:
+                #need to use relative path
+                name = fname.split('/')[0] + '/' + name
```

Comment:
I use it only with:
#include "/usr/share/gromacs/top/opls.ff/forcefield.itp"
in topol.top

this should help:
FillFileBuffer(os.path.basename(name), filebuffer, cwd, defines)

see here: https://github.com/MrTheodor/bakery/blob/master/src/gromacs_topology.py